ui/src/main/webapp/src/app/shared/upload/alternative-upload-queue.component.scss

```diff
@@ -26,6 +26,7 @@ li {
 .progress-bar {
   padding-top: 5px;
   padding-bottom: 5px;
+  color: #363636;
```

Comment:
@mrizzi if we have so different colors in empty vs full progress bars we need to change color based on the background dynamicly. 
If we could get rid of labels on progress bar then patternfly examples referenced by @klinki is good way to solve this issue. I am not sure how this change is in line with Marc's concept, but I would vote for that if dynamic change of font color is not too easy.